changelog.d/11749.doc

```diff
@@ -0,0 +1 @@
+Update room spec url in config files.
```

Comment:
Wasn't aware fixes like this to previous PR's would require changelogs, sorry. However, the changelog here doesn't seem related? :thinking:

Reply:
No worries: the linter is merciless. It wants a changelog for every PR.

Looks like I got my PR numbers wrong, argh. Thanks for spotting!